tsconfig.json

```diff
@@ -5,10 +5,10 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "lib": ["es6", "dom", "es2018.promise", "es2017"],
-    "module": "es6",
+    "module": "es2020",
```

Comment:
This is required to be able to do dynamic imports.

Reply:
@SleeplessByte Is there any downside to using `es2020` as the module system?

Reply:
Not really, because you'll be using babel to "downgrade" it again :). All good.